docs/helm/upgrades.md

```diff
@@ -10,6 +10,11 @@ Releases](https://github.com/Alfresco/acs-deployment/releases).
 Here follows a more detailed explanation of any breaking change grouped by
 version in which they have been released.
 
+## Unreleased
+
+* Search Enterprise is now the default search engine when installing Enterprise
```

Comment:
Can you add a small doc on how to configure an external ES index and link to it from here?